pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -73,17 +72,17 @@ func NewDefaultRegistry(args *RegistryArgs) framework.Registry {
 		volumebinding.Name: func(_ *runtime.Unknown, _ framework.FrameworkHandle) (framework.Plugin, error) {
 			return volumebinding.NewFromVolumeBinder(args.VolumeBinder), nil
 		},
-		volumerestrictions.Name:        volumerestrictions.New,
-		volumezone.Name:                volumezone.New,
-		nodevolumelimits.CSIName:       nodevolumelimits.NewCSI,
-		nodevolumelimits.EBSName:       nodevolumelimits.NewEBS,
-		nodevolumelimits.GCEPDName:     nodevolumelimits.NewGCEPD,
-		nodevolumelimits.AzureDiskName: nodevolumelimits.NewAzureDisk,
-		nodevolumelimits.CinderName:    nodevolumelimits.NewCinder,
-		interpodaffinity.Name:          interpodaffinity.New,
-		nodelabel.Name:                 nodelabel.New,
-		requestedtocapacityratio.Name:  requestedtocapacityratio.New,
-		serviceaffinity.Name:           serviceaffinity.New,
+		volumerestrictions.Name:                    volumerestrictions.New,
+		volumezone.Name:                            volumezone.New,
+		nodevolumelimits.CSIName:                   nodevolumelimits.NewCSI,
+		nodevolumelimits.EBSName:                   nodevolumelimits.NewEBS,
+		nodevolumelimits.GCEPDName:                 nodevolumelimits.NewGCEPD,
+		nodevolumelimits.AzureDiskName:             nodevolumelimits.NewAzureDisk,
+		nodevolumelimits.CinderName:                nodevolumelimits.NewCinder,
+		interpodaffinity.Name:                      interpodaffinity.New,
+		nodelabel.Name:                             nodelabel.New,
+		noderesources.RequestedToCapacityRatioName: noderesources.NewRequestedToCapacityRatio,
```

Comment:
nit: please move it to line 72 so that it is grouped with the other noderesources plugins.